src/Documentation/SidebarMenu/helper.js

```diff
@@ -0,0 +1,136 @@
+import startCase from 'lodash.startcase'
+
+import sidebar from '../sidebar'
+
+/*
+  We will use this function to normalize sidebar structure and create
+  all of the resurces we need to prevent future recalculations.
+
+  Target structure example:
+
+  {
+    label: "Add Files or Directories",
+    path: "/doc/get-started/add-files",
+    source: "/static/docs/get-started/add-files.md",
+    prev: "/doc/get-started/configure",
+    next: "/doc/get-started/share-data",
+    children: []
+  }
+*/
+
+const PATH_ROOT = '/doc/'
+const FILE_ROOT = '/static/docs/'
+const FILE_EXTENSION = '.md'
+
+let prevReference // We will save prev item reference here to use for the next and prev fields
```

Comment:
it's definitely > 80 symbols. I would recommend to install the pre-commit hook: https://dvc.org/doc/user-guide/contributing-documentation#development-environment

Reply:
and run the prettier from the command line once again to reformat what has been committed already

Reply:
I think prettier ignores comment lengths, so it probably didn't help. But I will fix comment lengths.

Reply:
you might be right - it just does not know what can be done with it.